nautobot/core/views/__init__.py

```diff
@@ -292,14 +293,18 @@ def get(self, request):
                     # For UIViewSet, view_func.cls gets what we need; for an ObjectListView, view_func.view_class is it.
                     view_or_viewset = getattr(view_func, "cls", getattr(view_func, "view_class", None))
                     queryset = view_or_viewset.queryset.restrict(request.user, "view")
+                    model = queryset.model
                     # For a UIViewSet, .filterset_class, for an ObjectListView, .filterset.
                     filterset = getattr(view_or_viewset, "filterset_class", getattr(view_or_viewset, "filterset", None))
                     # For a UIViewSet, .table_class, for an ObjectListView, .table.
                     table = getattr(view_or_viewset, "table_class", getattr(view_or_viewset, "table", None))
 
                     # Construct the results table for this object type
                     filtered_queryset = filterset({"q": form.cleaned_data["q"]}, queryset=queryset).qs
-                    table = table(filtered_queryset, orderable=False)
+                    if model == Prefix:
+                        table = table(filtered_queryset, hide_hierarchy_ui=True, orderable=False)
+                    else:
+                        table = table(filtered_queryset, orderable=False)
```

Comment:
Why's this need to be conditional? Aren't other models (Location, TenantGroup, etc.) also needing this fix? What breaks if we just always `hide_hierarchy_ui=True` here?